api/src/server.ts

```diff
@@ -52,6 +59,21 @@ const start = async () => {
       credentials: true,
     },
   });
+
+  io.on('connection', async socket => {
```

Comment:
This callback function doesn't have `await` in it, so `async` is not necessary.

Reply:
ok will amend it